TERMSOFSERVICE.md

```diff
@@ -308,7 +345,7 @@ any such matter without our prior written consent. We will use reasonable
 efforts to notify you of any such claim, action or proceeding upon becoming
 aware of it.
 
-# 13. Diclaimers and Limitation of Liability (except German and Austrian Users)
+# 13. Diclaimers and Limitation of Liability (except European Union Users)
```

Comment:
expand disclaimers and limitation of liability from German and Austrian to all European Union users.